test/presubmit-tests.sh

```diff
@@ -62,9 +61,6 @@ done
 
 TEST_RESULTS_GCS_DIR=gs://${TEST_RESULT_BUCKET}/${PULL_PULL_SHA}/${TEST_RESULT_FOLDER}
 ARTIFACT_DIR=$WORKSPACE/_artifacts
-WORKFLOW_COMPLETE_KEYWORD="completed=true"
-WORKFLOW_FAILED_KEYWORD="phase=Failed"
-PULL_ARGO_WORKFLOW_STATUS_MAX_ATTEMPT=$(expr $TIMEOUT_SECONDS / 20 )
```

Comment:
why remove?

Reply:
`WORKFLOW_COMPLETE_KEYWORD` and `WORKFLOW_FAILED_KEYWORD` are not longer needed since the completion checks are done differently. `PULL_ARGO_WORKFLOW_STATUS_MAX_ATTEMPT` can be replaced by a readable timeout.